arch/arm/soc/arm/mps2/soc.h

```diff
@@ -7,6 +7,12 @@
 #ifndef _SOC_H_
 #define _SOC_H_
 
+#if defined(CONFIG_SOC_MPS2_AN521)
+#define __SAUREGION_PRESENT       1U        /* SAU regions present */
+#define __FPU_PRESENT             CONFIG_CPU_HAS_FPU
+#define __DSP_PRESENT             1U        /* no DSP extension present */
```

Comment:
Here the comment does not match the value ('1' should indicate that the DSP is present)

Reply:
Good catch, will fix